azurerm/resource_arm_container_group.go

```diff
@@ -149,9 +149,20 @@ func resourceArmContainerGroup() *schema.Resource {
 							Type:         schema.TypeInt,
 							Optional:     true,
 							ForceNew:     true,
+							Deprecated:   "Use `ports` instead.",
 							ValidateFunc: validation.IntBetween(1, 65535),
 						},
 
+						"ports": {
+							Type:     schema.TypeList,
+							Optional: true,
+							ForceNew: true,
+							Elem: &schema.Schema{
+								Type:         schema.TypeInt,
+								ValidateFunc: validation.IntBetween(1, 65535),
```

Comment:
There is an existing validation function `validate.PortNumber` that could be used here

Reply:
I can't find it.

Reply:
It is in the [validate package](https://github.com/terraform-providers/terraform-provider-azurerm/blob/123d97cb20807ae534d5ee603c03fc33021da6a7/azurerm/helpers/validate/network.go#L73)